src/commands/transaction/list.ts

```diff
@@ -27,10 +27,34 @@ interface PromptAnswers {
   propertyId?: string;
 }
 
+const displayConciseTransactions = (response: SuccessResponse) => {
+  const transactions = response.data.payload as ListResponsePayload;
+  if (!transactions.length) {
+    return console.log(chalk.cyan("No transactions to display"));
+  }
+
+  const table = display.table.createCompact({
+    head: ["Trans ID", "Title", "Amount", "Property"],
+    colWidths: [28, 18, 10, 12],
+    colAligns: ["center", "left", "left", "left"],
+  });
+  transactions.forEach(
+    ({ id, title, amountChange, property }) => {
+      table.push([
+        id,
+        title,
+        display.values.formatPointsChange(amountChange),
+        property.name,
+      ]);
+    }
+  );
+  display.table.print(table);
+};
+
 const displayTransactions = (response: SuccessResponse) => {
   const transactions = response.data.payload as ListResponsePayload;
   if (!transactions.length) {
-    return console.log("Listed 0 transactions.");
+    return console.log(chalk.cyan("Listed 0 transactions."));
```

Comment:
Just this one `chalk.cyan` left - would you mind quickly fix this?

Reply:
Removed chalk.cyan. Think it should be good now. Please let me know if there is anything else.